bazel/PPROF.md

```diff
@@ -1,12 +1,10 @@
-# Memory consumption testing with `pprof`
+# CPU or memory consumption testing with `pprof`
 
 To use `pprof` to analyze performance and memory consumption in Envoy, you can
 use the built-in statically linked profiler, or dynamically link it in to a
 specific place yourself.
 
-# Linking
-
-## Static Linking
+## Collecting CPU or HEAP profile for a full execution of envoy
```

Comment:
Nit: HEAP is not an acronym, so I would just make this "heap"

Reply:
Done.